package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "15.1.0",
+  "version": "16.1.0",
```

```suggestion
  "version": "16.0.0",
```
Not sure if keeping the minor version at 1 was intentional.

---

Version was updating as other PRs merged, and realized this is not really a breaking change to the coronavirus-research schema so I am just incrementing the minor version number.  Thanks!